code/game/machinery/doors/airlock.dm

```diff
@@ -927,6 +928,20 @@
 		if(panel_open && detonated)
 			to_chat(user, "<span class='warning'>[src] has no maintenance panel!</span>")
 			return
+
+		if(whitelist_door)		//Whitelist doors kill you if you fucking open them. Crispy clerk.
+			if(ishuman(user))
+				var/mob/living/H = user
+				to_chat(H, "<span class='userdanger'>YOU ARE SHOCKED WITH THE FORCE OF 10,000 VOLTS!</span>")
+				to_chat(H, "<span class='notice'>This log will be sent to admins. Repeated attempts will result in a ban.</span>")
+				//It doesn't, I just fucking hate you.
+
+				//Tell everyone who you are
+				for(var/mob/M in GLOB.player_list)
+					to_chat(M, "<span class='userdanger'>[uppertext(user.real_name)] HAS BEEN SMITED BY THE HEAD.</span>")
+				H.apply_damage(999999999, BURN, BODY_ZONE_CHEST)	//Sick of fucking clerks ngl.
+				return
```

Comment:
This should have a warning on the door, I've already encountered at least 1 newer player that had no idea this was a thing and just wanted a fuckin mask and died for it. Also had several other people die because they saw a screwdriver next to a door and went "What's this?" Also don't lie about things that can get you banned, that's kinda just awful. Also you could literally just have kept this at this and still solve the issue.
```suggestion
		if(whitelist_door)
			if(ishuman(user))
				return
```

Reply:
Dude I've dealt with clerks and this bullshit constantly for the last 3 months. I'm going to keep the instakill but give it a warning message upon examine.

I'm not gonna sugarcoat it, they deserve to die.